src/screeninfo/mod.rs

```diff
@@ -153,7 +153,7 @@ impl ScreenInfo {
         }
 
         self.create_status_bar_window(event);
-        self.status_bar.as_mut().unwrap().draw();
+        self.status_bar.as_mut().unwrap().draw_without_border();
```

Comment:
makes sense but i dont really get why the statusbar didnt have any borders around it